### DISCUSSION TITLE
Implementing and comparing DCE techniques for WASM (22Q2 brainstorming)

### DISCUSSION THREAD

**Discussion by jsoares** [consensus-shipyard/consensuslab#140] [Project proposals]
- DCE threading support in wasmtime
- Writing/running benchmarks for threaded wasmtime
- Multiple gas routes based on the devs providing accurate read/write set and/or part of transactions expected to be under contention. If we can easily speed it up in a multicore- you pay less.

> We'll have to revisit scope for #11 and #12 

@quema @adlrocha @sergefdrv @LefKok

**Comment by sergefdrv** [comment]
@jsoares Could you please remind what is meant by "DCE threading" here?

**Reply by jsoares** [reply]
Deterministic Concurrent Execution; was too long for title :)

**Reply by quema** [reply]
You got it ;-)
